src/commands/_session_id.ts

```
@@ -0,0 +1,63 @@
+			log(JSON.stringify(null));
+		}
+		else {
+			logger.warn(`No session found with ID: ${params.id}`);
+		}
+		return { status: 'not_found' };
+	}
+
+	if (params.useJson) {
+		const payload = { sessionId: params.id, totalCost: sessionUsage.totalCost };
+		if (params.jq != null) {
+			const jqResult = await processWithJq(payload, params.jq);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)